packages/blockchain-wallet-v4-frontend/src/data/alerts/types.ts

```diff
@@ -1,7 +1,13 @@
 import { CoinType } from '@core/types'
-import { ToastNatureType } from 'blockchain-info-components'
 
 // types
+export enum ToastNatureType {
+  ERROR = 'error',
+  INFO = 'info',
+  SUCCESS = 'success',
+  WARN = 'warn'
+}
+
```

Comment:
This is because the `enum` seems to be deleted on runtime, so Webpack warns that it cannot find the exported variable.